dev/com.ibm.ws.microprofile.graphql.1.0/resources/com/ibm/ws/microprofile/graphql/resources/MPGraphQL.nlsprops

```diff
@@ -0,0 +1,30 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMGQ
+#COMPONENTNAMEFOR CWMGQ MicroProfile GraphQL API 
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by MP GraphQL Impl -  range is 0000-2999, plus 9999 for the emergency message
+#-----------------------------------------------------------------------------------------------------------------------------
+
+ERROR_GENERATING_SCHEMA_CWMGQ0001E=CWMGQ0001E: An error was detected that prevents the MicroProfile GraphQL application from starting in web module, {0}.
+ERROR_GENERATING_SCHEMA_CWMGQ0001E.explanation=A severe error was detected while processing the MP GraphQL application in the specified module. This error prevents the application from starting successfully.
```

Comment:
This explanation really just restates the description. Is there any more information we can offer about the kind of error or what might have caused it?

Reply:
Not really.  This error message is printed as a sort of "catch-all" for any error in the application (web module) that would prevent the app from starting.  There should be accompanying information in the logs that would indicate the source of the problem.

Reply:
ok